doc/source/whatsnew/v1.1.0.rst

```diff
@@ -98,6 +98,10 @@ Other enhancements
   This can be used to set a custom compression level, e.g.,
   ``df.to_csv(path, compression={'method': 'gzip', 'compresslevel': 1}``
   (:issue:`33196`)
+- :meth:`DataFrame.plot` will now allow the ``subplots`` parameter to be
+  a list of iterables specifying column groups, so that columns may be
+  grouped together in the same subplot. (:issue:`29944`)
```

Comment:
we usually put issue number here, not the PR number, so in this case, would use `29688`